src/widgets/slider/lv_slider.h

```diff
@@ -125,17 +138,31 @@ bool lv_slider_is_dragged(const lv_obj_t * obj);
 /**
  * Get the mode of the slider.
  * @param slider       pointer to a slider object
- * @return          see ::lv_slider_mode_t
+ * @return          see `lv_slider_mode_t`
  */
 lv_slider_mode_t lv_slider_get_mode(lv_obj_t * slider);
 
+/**
+ * Get the orientation of slider.
+ * @param obj       pointer to a slider object
+ * @return          slider orientation from `lv_slider_orientation_t`
+ */
+lv_slider_orientation_t lv_slider_get_orientation(lv_obj_t * slider);
+
 /**
  * Give the slider is in symmetrical mode or not
  * @param obj       pointer to slider object
  * @return          true: in symmetrical mode false : not in
 */
 bool lv_slider_is_symmetrical(lv_obj_t * obj);
 
+/**
+ * Set the orientation of slider.
+ * @param obj           pointer to a slider object
+ * @param orientation   slider  orientation from `lv_slider_orientation_t`
+ */
+void lv_slider_set_orientation(lv_obj_t * obj, lv_slider_orientation_t orientation);
```

Comment:
I've just noticed you added this prototype twice in this header file.

Reply:
Fixed, thanks!

Reply:
Looks like still has two of them :)

Could you move this commit to another PR too? It doesn't seem relevant to xml.

Reply:
Fixed again :crossed_fingers: